submissions/4zorbin/html-css-popup/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" >
+    <title>Hooli-pop</title>
```

Comment:
Metatag Viewport is mandatory for mobile markup.
https://serpstat.com/ru/blog/chto-takoe-metateg-viewport/
Could you please add it?
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`

Reply:
done